src/lib/uniforms-surfnet/src/SubscriptionField.tsx

```diff
@@ -240,15 +240,15 @@ function Subscription({
                 labelAppend={<EuiText size="m">{description}</EuiText>}
                 error={showInlineError ? errorMessage : false}
                 isInvalid={error}
-                id={id}
+                id={`subscription-formrow-${name}`}
                 fullWidth
             >
                 <div>
                     {!disabled && (
                         <>
                             <EuiButtonIcon
                                 className="reload-subscriptions-icon-button"
-                                id={`refresh-icon-${id}`}
```

Comment:
I assume that systemen-tests use the `refresh-icon-${id}` in tests

Weird that we use ${id} in a couple of places.

Reply:
When I revert it to `refresh-icon-${id}` the delta is

```
    -                                       "id": "subscription-formrow-x.0-refresh-icon",
    +                                       "id": "refresh-icon-uniforms-0000-001i",
```

Also a random id. If we don't make something stable they aren't usable in the system-tests anyway.

BTW I'm not sure if tthe random uniforms id is new (perhaps due to the EUI upgrade) or if it was already like this.